components/sync15/src/request.rs

```diff
@@ -455,13 +482,15 @@ where
     }
 }
 
+// A struct used for each piece of information uploaded
```

Comment:
Again struct is redundant, also 'piece' implies singular. This comment doesn't add value for me.